.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,7 +3,7 @@
 The following has been addressed in the PR:
 
 * [ ] There is a related issue
-* [ ] All code matches the [style guide](https://github.com/dojo/meta/blob/master/STYLE.md)
+* [ ] All code has been formatted with `prettier`
```

Comment:
Is this okay for the pull request template?

Reply:
I think this is fine.

Reply:
sounds ok to me. Unless you replace `with` with `via` ?